spark/v3.5/spark-extensions/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckViews.scala

```diff
@@ -33,9 +32,9 @@ object CheckViews extends (LogicalPlan => Unit) {
 
   override def apply(plan: LogicalPlan): Unit = {
     plan foreach {
-      case CreateIcebergView(ResolvedIdentifier(_: ViewCatalog, ident), _, query, columnAliases, _,
+      case CreateIcebergView(r@ResolvedIdentifier(_: ViewCatalog, _), _, query, columnAliases, _,
```

Comment:
nit: How about `resolvedIdentifier` instead of `r`

Reply:
makes sense, updated